src/native/corehost/apphost/standalone/CMakeLists.txt

```diff
@@ -9,7 +9,7 @@ set(DOTNET_PROJECT_NAME "apphost")
 # dependencies is not possible for some reason.
 # This cannot be enabled for MacOS (Darwin) since its RPATH works in a different way,
 # doesn't apply to libraries loaded via dlopen and most importantly, it is not transitive.
-if (NOT CLR_CMAKE_TARGET_OSX)
+if (NOT CLR_CMAKE_TARGET_OSX AND NOT CLR_CMAKE_TARGET_MACCATALYST AND NOT CLR_CMAKE_TARGET_IOS AND NOT CLR_CMAKE_TARGET_TVOS)
```

Comment:
We use this complex condition here and just `NOT CLR_CMAKE_TARGET_APPLE` in the file below. Seems like it should be unified. And actually, the host is never used for NativeAOT, so maybe these can be reverted too?

Reply:
Thanks yes I forgot this one.